tests/app/test_jinja_filters.py

```diff
@@ -18,16 +19,147 @@
     format_unit_input_label,
     get_currency_symbol,
     get_formatted_address,
-    get_formatted_currency,
     get_width_for_number,
     map_list_collector_config,
     map_summary_item_config,
     should_wrap_with_fieldset,
     strip_tags,
 )
 from app.questionnaire.questionnaire_schema import QuestionnaireSchema
+from app.utilities.decimal_places import get_formatted_currency
 from app.utilities.schema import load_schema_from_name
 
+TEST_FORMAT_CURRENCY_PARAMS = (
+    "value, currency, locale_string, decimal_limit, expected_result",
+    [
+        (Decimal("2.1"), "GBP", "en_GB", None, "£2"),
```

Comment:
As discussed in tech session, I don't think this is correct. This would cause problems with the previous transform scenario. If the value has a decimal, then the expected value should never be cropped. We can safely assume, if the value has a decimal, then they were allowed to enter that.
For this I would expect the answer to be £2.10

Reply:
This example would fail currently:

```py
def test_multiple_metadata_transform_format_currency_placeholder(
    mock_renderer, mock_schema, mock_location
):
    placeholder_list = [
        {
            "placeholder": "total",
            "transforms": [
                {
                    "transform": "add",
                    "arguments": {
                        "lhs": Decimal("1.23"),
                        "rhs": Decimal("1"),
                    },
                },
                {
                    "transform": "format_currency",
                    "arguments": {
                        "number": {"source": "previous_transform"}
                    },
                },
            ],
        }
    ]

    metadata = get_metadata()

    parser = PlaceholderParser(
        language="en",
        answer_store=AnswerStore(),
        list_store=ListStore(),
        metadata=metadata,
        response_metadata={},
        schema=mock_schema,
        renderer=mock_renderer,
        progress_store=ProgressStore(),
        location=mock_location,
        supplementary_data_store=SupplementaryDataStore(),
    )

    placeholders = parser(placeholder_list)

    assert placeholders["total"] == "£2.23"
```

```sh
tests/app/questionnaire/test_placeholder_parser.py:53 (test_multiple_metadata_transform_format_currency_placeholder)
'£2' != '£2.23'

Expected :'£2.23'
Actual   :'£2'
```

Reply:
Updated as discussed, and added the above test cases 👍